charts/incubator/misskey/values.yaml

```
@@ -0,0 +1,159 @@
+image:
+  repository: misskey/misskey
+  pullPolicy: IfNotPresent
+  tag: 12.119.0@sha256:a7beecb59b5089068463ca3847ac34dcb8445108da75ed8f56822f670053aa31
+
+
+securityContext:
+  readOnlyRootFilesystem: false
+  runAsNonRoot: false
+
+podSecurityContext:
+  runAsUser: 0
+  runAsGroup: 0
+  fsGroup: 33
+
+
+service:
+  main:
+    ports:
+      main:
+        port: 3000
+        targetPort: 3000
```

```suggestion
        port: 3003
```
Generally values on questions and values.yaml should be in sync
(unless it's needed to be different)